lambdas/url_access_patterns/lambda_for_getHash/index.js

```diff
@@ -0,0 +1,34 @@
+const AWS = require("aws-sdk");
+
+//AWS.config.loadFromPath("../../../keys.json");
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: message,
+  };
+}
+
+exports.getHash = async (event) =>{
+    try{
+        var params={
+            TableName:"V-Transfer",
+            IndexName:"FIND_FILE_BY_URLID",
+            KeyConditionExpression: "#PK= :pk",
+            ExpressionAttributeNames:{
+                "#PK": "GS1_PK",            
+            },
+            ExpressionAttributeValues:{
+                ':pk':`${event.path.url_id}`,                
+            },                                    
+        }        
+        const url_data = await dynamo.query(params).promise()        
+        const return_data=url_data.Items[0]? url_data.Items[0].hash:"";        
```

Comment:
where is the conditional check happening, like if the number of clicks is available or not, and if the URL is visible or not?

Reply:
Noted.